package.json

```diff
@@ -76,10 +78,18 @@
     "posttest": "[ -z \"$TRAVIS\" ] || codecov",
     "codecov": "codecov",
     "bench": "BENCHMARK=true npm run test",
-    "version": "node -e \"console.log(require('./package.json').version)\""
+    "version": "node -e \"console.log(require('./package.json').version)\"",
+    "lint-staged": "lint-staged"
   },
   "dependencies": {
     "is-browser": "^2.0.1",
     "murmurhash-js": "^1.0.0"
-  }
+  },
+  "lint-staged": {
```

Comment:
Or maybe `npm run lint` should stay as is, because we are all used to call it this way ...